README.md

```diff
@@ -742,6 +742,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [NuPIC](https://github.com/numenta/nupic) - Numenta Platform for Intelligent Computing.
 * [Pylearn2](https://github.com/lisa-lab/pylearn2) - A Machine Learning library based on [Theano](https://github.com/Theano/Theano).
 * [scikit-learn](http://scikit-learn.org/) - The most popular Python library for Machine Learning.
+* [surprise](http://surpriselib.com) - A scikit for building and analyzing recommender systems.
```

Comment:
http://surpriselib.com/ is down? I got "This site can’t be reached."

Reply:
Thank you guys! Willie Parish